### PR TITLE
Summary report

### PR DESCRIPTION
**Pull request by francylang**
## What's this PR do?
I've been working on trying to create a summary button that would send all the texts at once. In this PR I have gotten a little closer, I am able to send student summaries on individual button clicks. 

## Where should the reviewer start?

The student.js file is where most of the magic is happening.

### CODE REVIEW

File: lib/components/Student/Student.js [hunk 1]
```diff
@@ -21,7 +20,25 @@ class Student extends Component {
     // await textFetch(student, coreValue);
   }
 
-  //onclick of the summary button text all counts
+  async sendSummary() {
```
Comment (jenPlusPlus) [inline lib/components/Student/Student.js]: nice use of async/await! :)

Comment (lfinney) [inline lib/components/Student/Student.js]: Dat async, tho

File: lib/reducers/CoreValueReducer.js [hunk 2]
```diff
@@ -17,14 +17,11 @@ const characterReducer = (state = coreValueData, action) => {
...
-  //     points + acc;
-  //     return acc;
-  //   }, 0);
-  //
-  //   return reduced;
+  case 'SEND_SUMMARIES':
+    const pointValues = Object.values(state);
+    return pointValues.reduce((acc, points) => {
+      return points + acc;
+    }, 0);
```
Comment (lfinney) [inline lib/reducers/CoreValueReducer.js]: This is cool! I haven't seen a reduce in a reducer before. A reducerer, if you will.

Comment (francylang) [reply lib/reducers/CoreValueReducer.js]: <3

**Pull request merged**